docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04 as build
+
+RUN apt-get update && apt-get install -y curl git apt-transport-https && \
+    curl -sSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
+    echo "deb https://deb.nodesource.com/node_10.x xenial main" | tee /etc/apt/sources.list.d/nodesource.list && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    apt-get update && apt-get install -y nodejs yarn
+
+ADD . /app/src
+
+WORKDIR /app/src
+
+RUN yarn install
+RUN yarn build
```

Comment:
Can we remove sourcemaps after building the app?

```
RUN rm build/static/{css,js}/*.map
```

Reply:
Is it possible to make the yarn build process not generate them? The .js files reference the source maps, so ideally we just wouldn't generate them and they wouldn't be referenced with comments in the minified js files either.

Reply:
Not with create-react-app. It's not a problem files referencing sourcemap files, as those are only loaded when someone opens the web dev tools. Then, those calls would return 404s, which is no big deal.

Reply:
Gotcha. I'll add a rm of the maps then 👍.

Reply:
We chatted offline and decided this doesn't really matter because this is open source.